new-cdc/eventuate-local-java-cdc-connector-mysql-binlog/src/main/java/io/eventuate/local/mysql/binlog/MySqlBinaryLogClient.java

```diff
@@ -151,7 +139,33 @@ private void handleWriteRowsEvent(Event event, Optional<BinlogFileOffset> starti
       binlogEntryHandlers
```

Comment:
This would be simpler if it was written using a for/if statements.

Reply:
I will fix